docs/software/container-engine.md

```diff
@@ -50,13 +50,13 @@ Since the `ubuntu.toml` file is located in the [EDF search path][ref-ce-edf-sear
     The above terminal snippet demonstrates how to launch a containerized environment using Slurm with the `--environment` option.
     Click on the :fontawesome-solid-circle-plus: icon for information on each command.
 
-    ```bash
-    daint-ln002 > srun --environment=ubuntu --pty bash   # (1)
+    ```console
+    [daint-ln002]$ srun --environment=ubuntu --pty bash   # (1)
```

Comment:
The original docs in KB had very long prompts, so when the docs were ported I tried to shorten all of them to a single `>`.

In this case adding the hostname to the prompt is probably useful, because it provides additional context about whether the shell is on the login node / compute node.

Reply:
Yeah, I've tried not to modify the code blocks significantly, but here the prompt indeed makes sense.

The only thing I didn't mention: it seems like `daint-ln002$ ...` does _not_ trigger highlighting either, but (e.g.) `[daint-ln002]$` does, which is why I did more than just change `>` to `$`. I don't know what the exact rules are unfortunately, and I'm leaving that for future investigation.